src/index.html

```diff
@@ -20,8 +21,387 @@
       href="styles/index.scss"
     />
   </head>
+  <!-- #endregion -->
 
   <body>
-    <h1>Catalog</h1>
+    <!-- #region header -->
+    <header class="header">
+      <a
+        href="#"
+        class="logo"
+      >
+        <img
+          src="/src/images/logo.png"
+          alt="Moyo Logo"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              href="#apple"
+              class="nav__link is-active"
+            >
+              Apple
+            </a>
+
+            <a
+              href="#samsung"
+              class="nav__link"
+            >
+              Samsung
+            </a>
+
+            <a
+              href="#smartphones"
+              class="nav__link"
+            >
+              Smartphones
+            </a>
+
+            <a
+              href="#laptops&computers"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+
+            <a
+              href="#gadgets"
+              class="nav__link"
+            >
+              Gadgets
+            </a>
+
+            <a
+              href="#tablets"
+              class="nav__link"
+            >
+              Tablets
+            </a>
+
+            <a
+              href="#photo"
+              class="nav__link"
+            >
+              Photo
+            </a>
+
+            <a
+              href="#video"
+              class="nav__link"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <!-- #endregion -->
+    <!-- #region main -->
+    <main class="cards-catalog">
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <img
+          src="images/imac.jpeg"
+          class="card__image"
+          alt="iMac 27"
+        />
+
+        <h1 class="card__title">
+          APPLE A1419 iMac 27" Retina
+          <br />
+          5K Monoblock (MNED2UA/A)
+        </h1>
+
+        <h3 class="card__product-id">Product code: 195434</h3>
+
+        <div class="stars">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+
+          <p class="stars__star-review">Reviews: 5</p>
+        </div>
+
+        <div class="card__price">
+          <p class="card__price-value">Price:</p>
+
+          <p class="card__price-total">$2,199</p>
+        </div>
+        <a
+          href="#"
+          class="card__button"
+          data-qa="card-hover"
+        >
+          BUY
+        </a>
+      </div>
+      <div class="card">
```

Comment:
Add the blank line between the cards
```suggestion
      </div>
      
      <div class="card">
```